dev/integration_tests/flutter_driver_screenshot_test/test_driver/main_test.dart

```diff
@@ -19,18 +19,18 @@ Future<void> main() async {
   tearDownAll(() => driver.close());
 
   test('A page with an image screenshot', () async {
-
     final SerializableFinder imagePageListTile =
         find.byValueKey('image_page');
     await driver.waitFor(imagePageListTile);
     await driver.tap(imagePageListTile);
     await driver.waitFor(find.byValueKey('red_square_image'));
     await driver.waitUntilNoTransientCallbacks();
 
-    final DriverScreenShotTester tester = DriverScreenShotTester(testName: 'red_square_image', deviceModel: deviceModel, driver: driver);
-    final List<int> screenShot = await tester.getScreenshotAsBytes();
-    final bool compareResult = await tester.compareScreenshots(screenShot);
-    expect(compareResult, true);
+    await expectLater(
+      driver.screenshot(),
```

Comment:
nit: Can we add a comment or TODO linking to the related issues about the no-op test?

Reply:
Done!